app/views/admin/tags/index.html.haml

```diff
@@ -0,0 +1,39 @@
+- content_for :page_title do
+  = t('admin.tags.title')
+
+= form_with url: admin_tags_url, method: :get, class: :simple_form do |form|
+  .filters
+    .filter-subset.filter-subset--with-select
+      %strong= t('admin.tags.moderation.title')
+      .input.select.optional
+        = form.select :status,
+                      options_for_select(admin_tags_moderation_options, params[:status]),
+                      prompt: t('generic.all')
+
+    .filter-subset.filter-subset--with-select
+      %strong= t 'generic.order_by'
+      .input.select
+        = form.select :order,
+                      options_for_select([[t('admin.tags.newest'), 'newest'], [t('admin.tags.oldest'), 'oldest'], [t('admin.tags.pending_review'), 'pending_review']], params[:order])
+
+  .fields-group
+    .input.string.optional
+      = form.text_field :name,
+                        value: params[:name],
+                        class: 'string optional',
+                        placeholder: t('admin.tags.name')
+
+  .actions
+    %button.button= t('admin.tags.search')
+    = link_to t('admin.tags.reset'), admin_accounts_path, class: 'button negative'
+
+%hr.spacer/
+
+.batch-table
+  .batch-table__body
+    - if @tags.empty?
+      = nothing_here 'nothing-here--under-tabs'
+    - else
+      = render partial: 'admin/trends/tags/tag', collection: @tags, locals: { f: nil }
```

Comment:
Arguably maybe this partial should be moved here (`admin/tags`) and referenced from `admin/trends/tags`

The `f: nil` thing is simply to remove the checkboxes, since in this view we're not doing bulk actions right now, and tbh, I don't know if bulk actions would be desired here.